.github/workflows/periodic.yml

```diff
@@ -26,3 +26,40 @@ jobs:
           title: Broken link detected by CI
           content-filepath: .github/ISSUE_TEMPLATE/broken-link.md
           labels: automated, broken link
+
+  test-bundle-periodic:
+    name: Operator Bundle Test
```

Comment:
Since there's some coverage for this in Shipyard's scripts tests now, IMHO it might make sense for this to be a Flake Finder job (moved to that GHA workflow).